pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -5353,7 +5263,6 @@ broken-packages:
   - Haggressive
   - hahp
   - haiji
-  - hail
```

Comment:
All you need to do is remove this line from the `broken-packages` section.

Reply:
Oops, the rest slipped in due to a last test build.

Reply:
Actually, no. I need to update the PR's target branch.